src/styles/card-stars.scss

```diff
@@ -0,0 +1,46 @@
+.card-content__stars {
```

Comment:
it's a BEM-mix, other words it's a block that at the same time **is an element** of another block
So it should live in the `card-content.scss` together with other elements of the `card-content` block